docs/pages/develop/user-interface/app-icons.mdx

```diff
@@ -28,18 +28,15 @@ For an in-detail walkthrough, see the video below:
 
 ### Android
 
-{/* TODO: (@aman) Remove comments below when monochrome is released for the next SDK. */}
+Further customization of the Android icon is possible using the [`android.adaptiveIcon`](/versions/latest/config/app/#adaptiveicon) property, which will override both of the previously mentioned settings.
```

Comment:
We're mixing calling it 'property' here, and 'field' further down below a number of times.